### PR TITLE
feat: Symlink script for SDK docs

### PR DESCRIPTION
**Pull request by HazAT**

### CODE REVIEW

File: scripts/sdk-symlink [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+const fs = require("fs");
+const path = require("path");
+
+const args = process.argv;
+var myArgs = process.argv.slice(2);
+
+var symlinkRecursiveSync = function(src, dest) {
```
Comment (AbhiPrasad) [inline scripts/sdk-symlink:9]: I think we can use `const` or `let` everywhere, we enforce node 12 with volta

File: scripts/sdk-symlink [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+var myArgs = process.argv.slice(2);
+
+var symlinkRecursiveSync = function(src, dest) {    
+  var exists = fs.existsSync(src);
+  var stats = exists && fs.statSync(src);
+  var isDirectory = exists && stats.isDirectory();
+  if (isDirectory) {
+    fs.mkdirSync(dest);
+    console.log(`# - New Dir ${dest}`);
+    fs.readdirSync(src).forEach(function(childItemName) {
```
Comment (AbhiPrasad) [inline scripts/sdk-symlink:16]: We can also use arrow functions :)

File: scripts/sdk-symlink [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+const fs = require("fs");
+const path = require("path");
+
+const args = process.argv;
```
Comment (AbhiPrasad) [inline scripts/sdk-symlink:6]: unused.

File: scripts/sdk-symlink [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    ));
+  } else {
+    const target = path.relative(path.dirname(dest), src);    
+    console.log(`## - Symlink ${target} -> ${dest}`);
+    fs.symlinkSync(target, dest);
+  }
+};
+
+if (!myArgs[0] || !myArgs[1]) {
+    console.error('Please call this command with like `sdk:copy FROM TO`');
```
Comment (AbhiPrasad) [inline scripts/sdk-symlink:27]:
```suggestion
    console.error('Arguments incorrectly specified. Please call this command with `yarn sdk:copy FROM TO`');
```

File: src/components/sidebar.js [hunk 5]
```diff
@@ -85,8 +85,7 @@ const toTree = nodeList => {
   
   // When you write new docs and you want to replace the old "platforms" with the "sdks"
   // tree, just add the the child that should be replaced here
-  // TODO: We need to add folders from SDKs that don't exist in platforms yet here
-  const newSdkDocs = ['javascript'];
+  const newSdkDocs = ['javascript', 'angular'];
```
Comment (rhcarvalho) [inline src/components/sidebar.js:88]:
```suggestion
  const newSdkDocs = ['javascript'];
```

**Pull request merged**